src/Assignment5/MathApp/MathUtilis.cs

```diff
@@ -0,0 +1,76 @@
+﻿// <copyright file="MathUtilis.cs" company="PlaceholderCompany">
+// Copyright (c) PlaceholderCompany. All rights reserved.
+// </copyright>
+
+namespace MathApp
+{
+    /// <summary>
+    /// Has the methods for the mathematical operations.
+    /// </summary>
+    public static class MathUtilis
+    {
+        /// <summary>
+        /// Read Input gets numbers from user.
+        /// </summary>
+        public static void Readinput()
+        {
+            float number1, number2;
+            int choiceofoperation;
+            Console.WriteLine("Enter the first number:");
+            number1 = ValidatorsApp.Program.GetInput();
+            Console.WriteLine("Enter the second number:");
+            number2 = ValidatorsApp.Program.GetInput();
+            Console.WriteLine("The operations to be performed.\n 1.Addition\n 2.Subtraction\n 3.Multiplication\n 4.Division\n 5.Exit\n");
+            Console.WriteLine("Enter your choice of operation");
+            choiceofoperation = ValidatorsApp.Program.GetIntegerInput();
+            Choice.ChoiceofMathoperation(choiceofoperation, number1, number2);
+        }
+
+        /// <summary>
+        /// Adds two numbers and passes it to the display method.
+        /// </summary>
+        /// <param name="number1">The first number entered by user</param>
+        /// <param name="number2">The second number entered by user</param>
+        public static void Addition(float number1, float number2)
+        {
+            DisplayApp.Program.Display(number1 + number2);
```

Comment:
Instead of using DisplayApp namespace everywhere you can add it in using statement.

Reply:
Using statement is added.